Assets/Scrips/CameraMovement.cs

```diff
@@ -25,10 +29,27 @@ public class CameraMovement : MonoBehaviour {
 
 	private void followTargetX ()
 	{
-		var v = transform.position;
-		v.x = target.transform.position.x;
-		float velocity = playerVelocity.x == 0 ? smoothness : playerVelocity.x;
-		transform.position = Vector3.Lerp (transform.position, v, System.Math.Abs (velocity) * Time.deltaTime);
+		float distance = Mathf.Abs (target.transform.position.x - transform.position.x);
```

Comment:
With the type change to "target", it will have a reference to gameobject still, so just use "target.gameObject.transform..."